ansible/plugins/callback/atmo_logger.py.j2

```diff
@@ -0,0 +1,98 @@
+import os
+from datetime import datetime
+
+DOCUMENTATION = '''
+    callback: atmo_logger
+    type: notification
+    short_description: write playbook output to per-instance logs
+    version_added: 2.5
+    description:
+        - This plugin is modeled after the default playbook stdout (without the banners)
+        - It writes logs per-instance to `<log_path>/<username>/<instance_uuid>`
+    extends_documentation_fragment:
+      - default_callback
+    requirements:
+      - Whitelist in configuration
+      - `log_path` defined as a directory in `ansible.cfg` and writeable by the user running Ansible
+'''
+
+from ansible.plugins.callback.default import CallbackModule as CallbackModule_default
+from ansible.module_utils._text import to_text
+from ansible import constants as C
```

Comment:
Move all imports above `DOCUMENTATION`. Convention is 3rd part libraries first, a blank line then all our libraries (int this case nothing),.